util/cleanup-versions.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+# Helper script to remove stale versions (without upstream branches) from the
+# staging project semi-automatically (users need to confirm before deleting).
+
+set -e
+
+UTIL_DIR="$(dirname "${BASH_SOURCE[0]}")"
+source "${UTIL_DIR}/logging.sh"
+
+# A safety cutoff. Only versions last deployed more than 4 weeks ago may be
+# deleted.
+CUTOFF="-P4W"
+
+# This is a constant instead of an argument because production versions should
+# be deleted carefully and manually.
+PROJECT_ARG="--project=wptdashboard-staging"
+
+function cleanup() {
+  info "Cleaning stale versions of $1..."
+
+  local SERVICE_ARG="-s $1"
+  local versions_to_delete=()
+
+  for version in $( gcloud app versions list $PROJECT_ARG $SERVICE_ARG --filter="last_deployed_time.datetime<$CUTOFF" --format="value(id)" ); do
```

Comment:
Would be good to (also) list all versions _after_ the cutoff, and for any that match a branch name for a merged PR, consider that stale too. Thoughts?

Reply:
https://github.com/web-platform-tests/wpt.fyi/pull/527